### DISCUSSION TITLE
Q: Unique lookup values

### DISCUSSION THREAD

**Discussion by arminta7** [mdelobelle/metadatamenu#613] [Q&A]
Hello! 

I was wondering if you could help me with a lookup. I want to show the unique quarters that are associated with related days. Right now I only know how to show the quarter for each which gives me a lot of repeat values. How can I only show the unique values?

Thank you!

<img width="473" alt="Screenshot 2024-02-18 at 19 28 43" src="https://github.com/mdelobelle/metadatamenu/assets/7927356/821c7ca4-f38f-403b-a926-5cc2f57fedb8">

<img width="929" alt="Screenshot 2024-02-18 at 19 28 59" src="https://github.com/mdelobelle/metadatamenu/assets/7927356/98acb772-acd0-4a4a-b6d6-52c09428281c">

**Comment by WhiskeyJack96** [comment]
If you use a custom summarizing function you could apply more logic to the pages
```
return [...(new Map(pages.map(x => [x.quarter.path, x.quarter])).values())]
```
<img width="613" alt="image" src="https://github.com/mdelobelle/metadatamenu/assets/10688621/aba2fafa-d853-4476-827e-847e5687b16f">

**Comment by arminta7** [comment]
Hello! I tried your solution and this is what I got. Not quite working!

<img width="1112" alt="Screenshot 2024-02-21 at 17 31 56" src="https://github.com/mdelobelle/metadatamenu/assets/7927356/6cf3ced9-d614-4315-b136-109bce15d518">

**Comment by WhiskeyJack96** [comment]
Can you share a couple example note yaml sections and/or the property config?

**Comment by arminta7** [comment]
Is this what you mean?

<img width="973" alt="Screenshot 2024-02-22 at 13 55 27" src="https://github.com/mdelobelle/metadatamenu/assets/7927356/d301dbce-291b-407e-b5c4-623ee707ff9e">

<img width="926" alt="Screenshot 2024-02-22 at 13 52 50" src="https://github.com/mdelobelle/metadatamenu/assets/7927356/650c99fb-0be6-452f-8bc3-1427bebbbdc3">

<img width="456" alt="Screenshot 2024-02-22 at 13 53 17" src="https://github.com/mdelobelle/metadatamenu/assets/7927356/a927a0d5-91f6-4e75-ace7-92bf1bf59653">

**Comment by WhiskeyJack96** [comment]
With these notes in the root of my vault:

[March.md](https://github.com/mdelobelle/metadatamenu/files/14381298/March.md)
[W1.md](https://github.com/mdelobelle/metadatamenu/files/14381299/W1.md)
[W2.md](https://github.com/mdelobelle/metadatamenu/files/14381300/W2.md)

and this fileclass defined:
[testissue.md](https://github.com/mdelobelle/metadatamenu/files/14381302/testissue.md)

I get the desired behavior, but I'm not sure I have the notes set up quite the same as you 😅
